client/src/components/organisms/DeleteRequestGroupDialog.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FunctionComponent } from "react";
+
+import FormModal from "./FormModal";
+
+interface Props {
+    requestGroupName?: string;
+    handleClose: () => void;
+    onSubmit: (e: React.FormEvent) => void;
+    onCancel: () => void;
+    numRequests: number;
+}
+
+const DeleteRequestTypeDialog: FunctionComponent<Props> = (props: Props) => {
+    return (
+        <FormModal
+            className="delete-request-type-form"
+            submitButtonText="Confirm"
+            title="Delete Type"
+            handleClose={props.handleClose}
+            show={true}
+            onSubmit={props.onSubmit}
+            onCancel={props.onCancel}
+        >
+            <div className="delete-request-type-form-content">
+                <p>
+                    Are you sure you want to delete the group <b>&#34;{props.requestGroupName ?? "this"}&#34;</b>? This will delete the{" "}
```

Comment:
Little confused about the purpose of "this" here? What use case does this cover

Reply:
The ?? operator resolves the expression to "this" in the case props.requestGroupName is nullish

Reply:
Gotcha, it might make more sense to have "the group <groupName>" display if it isn't nullish and just "this" otherwise